azure-devops/projects/io-developer-portal/io-developer-portal-backend.tf

```diff
@@ -0,0 +1,161 @@
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-rw, azuredevops_serviceendpoint_azurerm.PROD-IO, azuredevops_project.project]
+
+  project_id = azuredevops_project.project.id
+  name       = "${var.io-developer-portal-backend.repository.name}.deploy"
+  path       = "\\${var.io-developer-portal-backend.repository.name}"
+
+  repository {
+    repo_type             = "GitHub"
+    repo_id               = "${var.io-developer-portal-backend.repository.organization}/${var.io-developer-portal-backend.repository.name}"
+    branch_name           = var.io-developer-portal-backend.repository.branch_name
+    yml_path              = "${var.io-developer-portal-backend.repository.pipelines_path}/deploy-pipelines.yml"
+    service_connection_id = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.id
+  }
+
+  variable {
+    name  = "GIT_EMAIL"
+    value = data.azurerm_key_vault_secret.key_vault_secret["io-azure-devops-github-EMAIL"].value
+  }
+
+  variable {
+    name  = "GIT_USERNAME"
+    value = data.azurerm_key_vault_secret.key_vault_secret["io-azure-devops-github-USERNAME"].value
+  }
+
+  variable {
+    name  = "GITHUB_CONNECTION"
+    value = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.service_endpoint_name
+  }
+
+  variable {
+    name  = "CACHE_VERSION_ID"
+    value = var.io-developer-portal-backend.pipeline.cache_version_id
+  }
+
+  variable {
+    name  = "PRODUCTION_AZURE_SUBSCRIPTION"
+    value = azuredevops_serviceendpoint_azurerm.PROD-IO.service_endpoint_name
+  }
+
+  variable {
+    name  = "PRODUCTION_APP_NAME"
+    value = var.io-developer-portal-backend.pipeline.production_app_name
+  }
+
+  variable {
+    name  = "PRODUCTION_RESOURCE_GROUP_NAME"
+    value = var.io-developer-portal-backend.pipeline.production_resource_group_name
+  }
+
+}
+
+# deploy serviceendpoint authorization
+resource "azuredevops_resource_authorization" "io-developer-portal-backend-deploy-github-ro-auth" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-ro, azuredevops_build_definition.io-developer-portal-backend-deploy, azuredevops_project.project]
+
+  project_id    = azuredevops_project.project.id
+  resource_id   = azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id
+  definition_id = azuredevops_build_definition.io-developer-portal-backend-deploy.id
+  authorized    = true
+  type          = "endpoint"
+}
+
+resource "azuredevops_resource_authorization" "io-developer-portal-backend-deploy-github-rw-auth" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-rw, azuredevops_build_definition.io-developer-portal-backend-deploy, azuredevops_project.project]
+
+  project_id    = azuredevops_project.project.id
+  resource_id   = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.id
+  definition_id = azuredevops_build_definition.io-developer-portal-backend-deploy.id
+  authorized    = true
+  type          = "endpoint"
+}
```

Comment:
@uolter I changed this because we need to add `io-azure-devops-github-ro` and `io-azure-devops-github-rw`.
Pipelines need this user `io-azure-devops-github-ro` to access to https://github.com/pagopa/azure-pipeline-templates

Reply:
I'll propagate this changes in all other projects